File: Allamvizsga/Allamvizsga/Scripts/Services/ServiceModel.js
function ServiceModel(data) {
    this.id = ko.observable(null);
    this.vin= ko.observable(null);
    this.owner=null;
    this.identifier = ko.observable(null);
    this.serviceDate = ko.observable(null);
    this.price = ko.observable(null);
    this.actualKm = ko.observable(null);
    this.flag = ko.observable(null);
    this.serviceId = ko.observable(null);
    this.historyes = ko.observableArray(null);
    
    
   if (data != null) {

       var historyes = _.map(data.Car, function (history, index) {
           return new RepaireModel(history);

       });


        this.historyes(historyes);
        this.id(data.ID)
        this.vin(data.VIN);
        this.owner = new OwnerModel(data.Owner);
        this.identifier(data.Identifier);
        this.serviceDate(data.ServiceDate);
        this.price(data.Price);
        this.actualKm(data.ActualKm);
        this.flag(data.Flag);
        this.serviceId(data.ServiceId);
    }
    
 }

File: Allamvizsga/Allamvizsga/Models/ServiceModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Allamvizsga.Models
{
    public class ServiceModel
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        public string Identifier { get; set; }
        public string VIN { get; set; }
        public int Price { get; set; }
        public int Flag { get; set; }
        public string PhoneNumber { get; set; }
        [ForeignKey("PhoneNumber")]
        public virtual OwnerModel Owner { get; set; }
        public int ServiceId { get; set; }
        [ForeignKey("ServiceId")]
        public virtual UserModel User { get; set; }
        public virtual ICollection<HistoryModel> Car { get; set; }

    }
}